test/test_db.cpp

```diff
@@ -37,6 +37,4 @@ TEST(DB_getCoreFiles_adding_path)
     CHECK_EQUAL(DB::get_core_file(path, DB::CoreFileType::Management), path + extension_management);
     CHECK_EQUAL(DB::get_core_file(path, DB::CoreFileType::Note), path + extension_note);
     CHECK_EQUAL(DB::get_core_file(path, DB::CoreFileType::Log), path + extension_log);
-    CHECK_EQUAL(DB::get_core_file(path, DB::CoreFileType::LogA), path + extension_log_a);
```

Comment:
extension_log_a and extension_log_b (lines 32 and 33) can be deleted too I suppose.

Reply:
Done.